pkg/controller/commonwebuiservice/commonwebuiservice_controller.go

```diff
@@ -366,12 +366,8 @@ func (r *ReconcileCommonWebUI) newDaemonSetForCR(instance *operatorsv1alpha1.Com
 	commonwebuiContainer := res.CommonContainer
 	commonwebuiContainer.Image = image
 	commonwebuiContainer.Name = res.DaemonSetName
-	commonwebuiContainer.Env[1].Value = instance.Spec.GlobalUIConfig.RouterURL
```

Comment:
Where are you setting this values now that they don't come from the CR? Does the container template have the default values that don't change?

Reply:
@ericabr earlier those values were set to defaults in the container and also were dynamically set from the cr in the controller at the same time.
Now they are still set as defaults values in the container, and since I cleaned up the cr, I took it out of the controller as well. There is no need for dynamically setting it anymore